molgenis-navigator/src/main/frontend/src/components/Navigator.vue

```diff
@@ -74,10 +74,10 @@
 
 <script>
   import _ from 'lodash'
-  import { QUERY_PACKAGES, QUERY_ENTITIES, RESET_STATE, GET_STATE_FOR_PACKAGE } from '../store/actions'
-  import { SET_QUERY, SET_ERROR, RESET_PATH, SET_PACKAGES } from '../store/mutations'
-  import { Package, INITIAL_STATE } from '../store/state'
-  import { mapState } from 'vuex'
```

Comment:
These formatting changes do not happen in my IntelliJ settings, do you have the javascript default setting enabled?